src/components/common/adi3-debrief-card-box/adi3-debrief-card-box.html

```diff
@@ -4,7 +4,7 @@
         <ion-row>
             <ion-col size="10" class="score-padding">
                 <div class="counter-background">
-                    <span class="count">{{item.q.score ? item.q.score : '-'}}</span>
+                    <span class="count">{{item.q.score ? item.q.score : '0'}}</span>
```

Comment:
Not sure if this is quite right, ```A score of 0 should be displayed as '0'```.

This would mean nulls would also show as ```0```.

Think it needs to be
```
<span class="count">{{item.q.score !== null ? item.q.score : '-'}}</span>
```
So if score is null it will show '-' otherwise the correct value